pylintrc

```diff
@@ -0,0 +1,3 @@
+[BASIC]
```

Comment:
Well, it's your local pylint. Committing this file if similar to committing some IDE files.
If you want to make things serious, explore https://travis-ci.org/
[StackOverflow question](https://stackoverflow.com/questions/12450348/how-to-integrate-any-python-lint-with-github-commit-status-api) about PyLint + GitHub.